tensorflow/tensorflow.bzl

```diff
@@ -403,6 +403,57 @@ def tf_gen_op_wrapper_py(name,
       ],)
 
 
+# Given a list of "ops_libs" (a list of files in the core/ops directory
```

Comment:
I've put it there only because similar functions for C++ and Python wrappers are already in this file, even if there is no apparent reason for having them as such a high level. I'm pleased to move it back down like you propose.